_includes/defined-schema/class-level-permissions.md

```diff
@@ -0,0 +1,59 @@
+# Class Level Permissions
+
+Setting Class level permissions through Defined Schema is a good first step into security systems available on Parse Server.
+
+## CLP Keys
+
+These CLP keys are available:
+
+- `find`: Control search permissions
+- `get`: Control direct ID get permission
+- `count`: Control counting objects permission
+- `create`: Create permission
+- `update`: Update permission
+- `delete`: Delete permission
+- `protectedFields`: Control get permission at field level
+
+You can set each CLP field with options to add a first strong security layer. This security layer will be applied on the Parse Class and all Parse Objects into this class.
+
+Note: If you update CLP you do not need to update Parse Objects. CLP is a security layer at Class Level not Object Level. For Object Level permission you can take a look to ALCs. You can use CLPs combined to ACLs to deeply secure your Parse Server.
+
+## CLP Key Options
```

Comment:
I would suggest replacing globally `key` with either `parameter` or `attribute` depending on the context.

Currently `parameter` seems to be more generalized as config keys

https://docs.parseplatform.org/parse-server/guide/#usage